docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -625,4 +668,149 @@ arming bypass is on for DroneCAN, and when it is set to *Normal Arming*, DroneCA
     :width: 60%
     :alt: DroneCAN Bypass Arming Parameter
 
-    DroneCAN Bypass Arming Parameter in IQ Control Center
\ No newline at end of file
+    DroneCAN Bypass Arming Parameter in IQ Control Center
+
+.. _dronecan_plug_and_play:
+
+*************************************
+Advanced Plug-And-Play Functionality
+*************************************
+Vertiq's DroneCAN nodes support both auto-bitrate detection as well as Dynamic Node Allocation. The following sections walk you through configuring your modules to use 
+both of these settings, as well as examples of the features in use with both the DroneCAN GUI tool as well as a flight controller.
+
+Auto-Bitrate Detection
+===========================
+As the name implies, auto-bitrate detection allows your Vertiq module to connect with an active DroneCAN bus, and automatically detect the bitrate in use. This allows 
+the module to configure its own bitrate accordingly without user intervention.
+
+In order to configure your module to use auto-bitrate detection, simply configure its ``DroneCAN Bitrate`` parameter to ``0``.
+
+.. note:: 
+	``DroneCAN Bitrate`` can be configured in three ways: :ref:`IQ Control Center <control_center_start_guide>`, DroneCAN configuration via the 
+	:ref:`DroneCAN GUI tool <dronecan_gui_basics>`, or through any of :ref:`Veritq’s APIs <getting_started_with_apis>`.
```

Comment:
DroneCAN configuration is also available in QGroundControl and MissionPlanner these days. Its ok to show the DroneCAN GUI Tool one as an example, but maybe just say its more generally available as a DroneCAN parameter